routes/api/v1/foods.js

```
@@ -44,4 +44,29 @@ router.get('/:id', async function(req, res, next){
   }
 })
 
+router.patch('/:id', async function(req, res, next){
+  try {
+    let food = await Food.findByPk(req.params.id)
+    let name = req.body.food.name
+    let calories = req.body.food.calories
+    if(name, calories == undefined ) {
```

I think this should be: ` if (name == undefined || calories == undefined)`

Tried this syntax out in a repl, and didn't seem to work.

Of note, the test only tries not passing in calories, which is maybe why this appears to work

---

This has been corrected. Indeed, my attempt at being fancy failed silently.